<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWidget</class>
 <widget class="QWidget" name="HomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonCompanyInfo">
          <property name="text">
           <string>公司简介</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStuffManage">
          <property name="text">
           <string>材料管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCarInfo">
          <property name="text">
           <string>车辆介绍</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClientManage">
          <property name="text">
           <string>客户管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFix">
          <property name="text">
           <string>维修记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOptLog">
          <property name="text">
           <string>操作日志</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonwarning">
          <property name="text">
           <string>系统告警</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSysSetting">
          <property name="text">
           <string>系统设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonUserManage">
          <property name="text">
           <string>用户管理</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>208</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExitLogin">
          <property name="text">
           <string>注销</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
